src/modules/cacheClient/cacheClient.service.ts

```diff
@@ -281,29 +293,22 @@ export class CacheClient implements ICacheClient {
     }
 
     private async refreshToken(): Promise<{ token: string; refreshToken: string }> {
-        try {
-            const { data } = await this.httpClient.get<{ token: string; refreshToken: string }>(
-                `/refresh_token${
-                    executionEnvironment() === ExecutionEnvironment.BROWSER
-                        ? ""
-                        : `?refresh_token=${this.refresh_token}`
-                }`,
-            );
-            return data;
-        } catch {
-            const pubKeyAndIdentityToken = await this._signerService.publicKeyAndIdentityToken();
-            const {
-                data: { refreshToken, token },
-            } = await this.httpClient.post<{ token: string; refreshToken: string }>("/login", {
-                identityToken: pubKeyAndIdentityToken.identityToken,
-            });
+        const { data } = await this.httpClient.get<{ token: string; refreshToken: string }>(
+            `/refresh_token${this.isBrowser ? "" : `?refresh_token=${this.refresh_token}`}`,
+        );
+        return data;
+    }
 
-            if (!this.isBrowser) {
-                this.httpClient.defaults.headers.common["Authorization"] = `Bearer ${token}`;
-                this.refresh_token = refreshToken;
-            }
-            this.pubKeyAndIdentityToken = pubKeyAndIdentityToken;
-            return { token, refreshToken };
+    /**
+     * @description Checks that auth token has been created, has not expired and corresponds to signer
+     * @todo specific endpoint on cache server to return login info instead of error
+     */
+    private async isAuthenticated(): Promise<boolean> {
+        try {
+            await this.getOwnedAssets(this._signerService.did);
```

Comment:
I think it might be good to actually use the `TEST_LOGIN_ENDPOINT` value here. Otherwise, one might forget to change `TEST_LOGIN_ENDPOINT` when they accomplish the `@todo` of using a dedicated endpoint.